integrationtests/security-it/src/test/resources/arquillian.xml

```diff
@@ -74,6 +74,43 @@
          </configuration>
       </container>
       
+      <container qualifier="simple-auth-node0" mode="manual">
+         <protocol type="jmx-as7">
+            <property name="executionType">REMOTE</property>
+         </protocol>
+         <configuration>
+            <property name="jbossHome">${basedir}/target/node0/wildfly-${version.wildfly}</property>
+            <!-- Needed for security domain configuration -->
+            <property name="serverConfig">standalone-test.xml</property>
+            <property name="javaVmArguments">-Djboss.node.name=node0
+                                             -Djboss.socket.binding.port-offset=100 
+                                             -Djava.net.preferIPv4Stack=true 
+                                             -Dsasl.credentials.properties=${basedir}/target/test-classes/jgroups-simple-auth.properties
+                                             -Dsasl.local.principal=test_user
+                                             -Dsasl.realm=test_realm
+            </property>
+            <property name="managementPort">10090</property>
+         </configuration>
+      </container>
+      
+      <container qualifier="simple-auth-node1" mode="manual">
+         <protocol type="jmx-as7">
```

Comment:
Should use the `Servlet 3.0` protocol I think: https://issues.jboss.org/browse/ISPN-5597